Cristianjs93/back-app-restaurant
# Feature/seeder

## Create Seeder for Populating Users, Restaurants, and Reviews

### Description
This pull request introduces a new seeder script to populate the `users`, `restaurants`, and `reviews` tables in the database using the Faker.js library. The seeder script generates realistic random data to facilitate testing and development.

### Changes Made
- Created a new seeder script named `seed.ts` inside the `prisma` folder.
- Created a new seeder file in each api folder.
- Utilized the Faker.js library to generate random data for users, restaurants, and reviews.
- Defined functions to create and insert mock data into the respective tables.
- Enhanced data variety by using various properties provided by Faker.js.

### How to Test
1. Pull the changes and type npm i in your computer's terminal
2. Modify the .seeder files of each api with the mock data you want to see.
3. Run the seeder script using the command: `npx prisma db seed`.
4. Check the database to confirm that the `users`, `restaurants`, and `reviews` tables have been populated with mock data.

### Additional Notes
- The seeder script provides a convenient way to populate the database with test data for development and testing purposes.
- The data generated by Faker.js is randomized and does not represent real user, restaurant, or review data.
